pkg/etcd/etcd.go

```diff
@@ -495,8 +496,16 @@ func (e *ETCD) removePeer(ctx context.Context, id, address string) error {
 				return err
 			}
 			if u.Hostname() == address {
+				if e.address == address {
+					continue
```

Comment:
If we're the node getting removed from the cluster, should we exit here? What is the point in staying running if we're evicted from the cluster. Won't we just crash when etcd shuts down?

Reply:
yeah its valid to do that, wont that affect the logic that you put for checking if the server stops and prints the message for deletion, we also will not be 100% sure that the deletion will take place and succeed it might fail